components/Moola.vue

```diff
@@ -103,13 +103,18 @@
                 return split.join('.');
             },
 
-            checkDecimals(event) {
+            blur(event) {
                 var val = this.removeCommas(event.target.value);
                 var split = val.split('.');
                 var el = this.$refs['moola'];
 
-                el.value = this.addCommas(Number(val).toFixed(this.precision));
-                this.$emit('input', Number(val).toFixed(this.precision));
+                var emitVal = Number(val).toFixed(this.precision);
+
+                if (isNaN(emitVal) || !isFinite(emitVal)) {
+                    emitVal = "0.00";
+                }
+                el.value = this.addCommas(emitVal);
```

Comment:
This should probably be renamed to `addDelimiters`

Reply:
Changed!